### PR TITLE
Merge Samsung v2.1 submission into submission-v2.1 

### PR DESCRIPTION
**Pull request by AhmedTElthakeb**

### CODE REVIEW

**Comment by freedomtan** [comment]
bring back new keyword settings from master to v2.1 by cherrypicking some patches from the master branch?
@AhmedTElthakeb to change Exynos backend backed to v2.1 submission settings

File: mobile_back_samsung/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Building MLPerf Open app with Samsung backend
+
+<!-- markdown-link-check-disable-next-line -->
+1. Update the "lib" folder at 'mobile_app_open/mobile_back_samsung/samsung/lib' by copying it from [here](https://github.com/mlcommons/mobile_back_samsung/tree/samsung_backend_flutter_libs/samsung_libs/mobile_back_samsung/samsung)
```
Comment (anhappdev) [inline mobile_back_samsung/README.md:4]: I would be helpful if Samsung can provide a checksum for each `*.so` file, which needs to be copied over, to verify its compatibility with the code.

Comment (AhmedTElthakeb) [reply mobile_back_samsung/README.md:4]: Updated the README with .so libs hashes. 
https://github.com/mlcommons/mobile_app_open/blob/submission_v2.1_revert_samsung/mobile_back_samsung/README.md

**Comment by AhmedTElthakeb** [comment]
Created new PR https://github.com/mlcommons/mobile_app_open/pull/606

**Pull request closed without merging**